recipes/crow/all/conanfile.py

```diff
@@ -0,0 +1,44 @@
+from conans import ConanFile, tools, CMake
+import os
+
+class CrowConan(ConanFile):
+    name = "crow"
+    homepage = "https://github.com/ipkn/crow"
+    description = "Crow is C++ microframework for web. (inspired by Python Flask)"
+    topics = ("conan", "web", "microframework", "header-only")
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "compiler", "arch", "build_type"
+    exports_sources = ["patches/*"]
+    license = "BSD-3-Clause"
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def requirements(self):
+        self.requires("boost/1.69.0")
```

Comment:
why old version?

Reply:
Unfortunately there's an open issue https://github.com/ipkn/crow/issues/374

Boost 1.70 introduces changes but the library hasn't been updated yet